configs/sst_virtualization-all.yaml

```diff
@@ -82,3 +82,16 @@ data:
           description: Add support for Windows guests to virt-v2v and virt-p2v
           dependencies:
             - libguestfs
+    # Package in RHEL and Fedora is different
+    - srpm_name: trustee-guest-components
+      build_dependencies:
+        - rust-toolset
+        - /usr/bin/pkg-config
```

Comment:
Let's not use file dependencies in Content Resolver; in this case, this is common enough that I think it can just be omitted here.

Reply:
fixed